### PR TITLE
Fix project name sorting

### PR DESCRIPTION
**Pull request by apmasell**
JIRA: GLT-1538

### CODE REVIEW

File: miso-web/src/main/webapp/scripts/project_ajax.js [hunk 1]
```diff
@@ -193,7 +192,7 @@ Project.ui = {
             "bJQueryUI": true,
             "iDisplayLength": 25,
             "aaSorting": [
-              [0, "desc"]
+              [0, "asc"]
```
Comment (callunity) [inline miso-web/src/main/webapp/scripts/project_ajax.js]: Why the change to `asc`? I would think that the lab would want to see more recent projects first.

Comment (apmasell) [inline miso-web/src/main/webapp/scripts/project_ajax.js]: Fair. It looked really weird the other way, but I guess we sort everything else by date.
